src/homework/day8/collections/MyBirds.java

```diff
@@ -0,0 +1,52 @@
+package homework.day8.collections;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MyBirds {
+
+
+    public static int countNumberOfVowels(String word) {
+
+        int counter = 0;
+        Pattern pattern = Pattern.compile("[аеёиоуыэюя]", Pattern.CASE_INSENSITIVE);
+        Matcher matcher = pattern.matcher(word);
+        while (matcher.find()) {
+            counter++;
+        }
+        return counter;
+
```

Comment:
Extra new line. Remove. The same for such cases.

Reply:
Extra line is removed.